scannerazuresc.properties

```diff
@@ -6,8 +6,13 @@ organizationUrl=https://www.sonarsource.com
 homepageUrl=https://docs.sonarsource.com/sonarcloud/advanced-setup/ci-based-analysis/sonarcloud-extension-for-azure-devops/
 issueTrackerUrl=https://sonarsource.atlassian.net/jira/software/c/projects/SONARAZDO/issues
 sourcesUrl=https://github.com/SonarSource/sonar-scanner-azdo
-archivedVersions=1.44.0,1.44.1,1.44.2,1.45.0,1.46.0,2.0.0,2.0.1,2.2.0,3.0.0,3.0.1,3.0.2,3.0.3,3.0.4,3.1.0,3.1.1,3.2
-publicVersions=3.3
+archivedVersions=1.44.0,1.44.1,1.44.2,1.45.0,1.46.0,2.0.0,2.0.1,2.2.0,3.0.0,3.0.1,3.0.2,3.0.3,3.0.4,3.1.0,3.1.1,3.2,3.3
+publicVersions=3.4.1
+
+3.4.1.description=Update tasks to Node 20+
+3.4.1.date=2025-08-7
```

Comment:
Hi @joan-biel-sonarsource, I noticed something when updating the official documentation.
Line 13 should read: `3.4.1.date=2025-08-07`
- It's missing the `0` in the month.

It's not consequential at this point and can be fixed in the next release (if needed).